app/views/propertyrepresentation/manage/myAgents.scala.html

```diff
@@ -42,7 +42,7 @@ <h1 class="govuk-heading-l">@messages("propertyRepresentation.showAgents.title")
                     content = Text(messages("propertyRepresentation.showAgents.properties", agent.propertyCount, propertyLinkingCount))
                 ),
                 TableRow(
-                    content = HtmlContent(Html(messages("propertyRepresentation.showAgents.viewAgentDProperties", s"${controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url}")))
+                    content = HtmlContent(Html(messages("propertyRepresentation.showAgents.viewAgentDProperties", s"${controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url}", s"View agent ${agent.name} (${agent.representativeCode})")))
```

Comment:
either move `View agent` text into the messages file pls (text should be in messages file)
so
```
s"${agent.name} (${agent.representativeCode})
```